code/modules/mob/living/carbon/human/species.dm

```diff
@@ -731,6 +735,65 @@
 	cold_level_2 = ICE_COLONY_TEMPERATURE - 40
 	cold_level_3 = ICE_COLONY_TEMPERATURE - 80
 
+/datum/species/catslug
+	name = "Catslug"
+	name_plural = "Catslugs"
+	icobase = 'icons/mob/human_races/r_catslug.dmi'
+	damage_mask_icon = 'icons/mob/dam_mask_catslug.dmi'
+	brute_damage_icon_state = "newt_brute"
+	burn_damage_icon_state = "newt_burn"
+	eyes = "blank_eyes"
+	unarmed_type = /datum/unarmed_attack/bite
+	slowdown = -1.25 //faster then the average human, can't wear shoes
+	total_health = 75 //small size, small health
+	burn_mod = 0.5 //slug coating, slippery
+	brute_mod = 0.8 //some innate brute defense so they have some protection
```

Comment:
less health but then they just have sythn dam reduction anyway to make up for it